drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/test/java/org/drools/workbench/screens/dtablexls/backend/server/conversion/DecisionTableXLSToDecisionTableGuidedConverterTest.java

```diff
@@ -260,6 +267,52 @@ public void checkConversionOfXLSXFiles() {
                                 any(String.class));
     }
 
+    @Test
+    public void checkConversionOfXLSXFilesRHDM1159() {
+        final InputStream is = this.getClass().getResourceAsStream("RHDM-1159.xls");
+        when(ioService.newInputStream(any(org.uberfire.java.nio.file.Path.class))).thenReturn(is);
+        final ConversionResult result = converter.convert(path);
+        assertNotNull(result);
+        final List<ConversionMessage> messages = result.getMessages();
+        assertEquals(6,
+                     messages.size());
+        assertTrue(messages.get(0).getMessage().startsWith("Created Import"));
+        assertTrue(messages.get(1).getMessage().startsWith("Created Import"));
+        assertTrue(messages.get(2).getMessage().startsWith("Created Import"));
+        assertTrue(messages.get(3).getMessage().startsWith("Created Import"));
+        assertTrue(messages.get(4).getMessage().startsWith("Created Import"));
+        assertTrue(messages.get(5).getMessage().startsWith("Created Guided Decision Table 'SampleAUWRule"));
```

Comment:
I got just one message in runtime , is that ok?
![Screenshot from 2020-01-22 09-46-13](https://user-images.githubusercontent.com/8044780/72878877-15007c80-3cfc-11ea-88ea-4d881a2af013.png)

Reply:
It's explainable - the _real_ project defines imports (for the DO's in the XLS) whereas the test did not mock the presence of such imports. I have updated the test to match what is observed when running "for real".